src/UpdateLocalizedResources.targets

```diff
@@ -0,0 +1,108 @@
+
+  <!-- Emit the localized EmbeddedResource items before the common targets start using them -->
+  <PropertyGroup>
+    <PrepareResourceNamesDependsOn>
+      EmitLocalizedResources;
+      UpdateLocalizedResxForEnglishResx;
+      $(PrepareResourceNamesDependsOn)
+    </PrepareResourceNamesDependsOn>
+  </PropertyGroup>
+
+  <Target Name="EmitLocalizedResources">
+    <ItemGroup>
+      <NeutralResources Include="@(EmbeddedResource)" Condition="'%(Extension)' == '.resx'"/> 
+    </ItemGroup>
+
+    <EmitLocalizedResources 
+      NeutralResources="@(NeutralResources -> '%(FullPath)')"
+      AssemblyName="$(AssemblyName)"
+      LocalizedResxRoot="$(IntermediateOutputPath)"
+      RelativePathToXlfRoot=".\xlf\">
+
+      <Output TaskParameter="ResolvedXlfResources" ItemName="Xlf_Temp" />
+      <Output TaskParameter="ResolvedLocalizedResxResources" ItemName="Resx_temp" />
+
+    </EmitLocalizedResources>
+
+    <ItemGroup>
+      <XlfFiles Include="@(Xlf_Temp)"/>
+      <EmbeddedResource Include="@(Resx_temp)"/>
+    </ItemGroup>
+
+  </Target>
+
+    <!-- The default english resources do not have corresponding xlf files. Simply copy them and add en in the name -->
+    <!-- TODO: Maybe create a dummy xlf file for English so we can delete this target? -->
+  <Target Name="UpdateLocalizedResxForEnglishResx">
+
+      <ItemGroup>
+        <EnglishResources Include="@(NeutralResources -> '$(IntermediateOutputPath)%(Filename).en%(Extension)')"/>
+      </ItemGroup>
+
+      <Copy
+        SourceFiles="@(NeutralResources)"
+        DestinationFiles="@(EnglishResources)"
+      />
+
+      <ItemGroup>
+        <EmbeddedResource Include="@(EnglishResources)">
+          <LogicalName>$(AssemblyName).%(FileName).resources</LogicalName>
+        </EmbeddedResource>
+      </ItemGroup>      
+
+  </Target>
+
+  <Target Name="UpdateXlf"
+          BeforeTargets="CoreResGen"
+          AfterTargets="SplitResourcesByCulture"
+          DependsOnTargets="EmitLocalizedResources;UpdateLocalizedResxForEnglishResx" 
+          Condition="$(LocalizedBuild) == 'true' and $(IsTestProject) != 'true'">
+    
+    <UpdateXlfFromResx
+      Condition="%(Identity) != ''"
+      ResxPath="%(NeutralResx)"
+      XlfPath="@(XlfFiles)"
+    />
+
+  </Target>
+
+  <Target Name="ConvertXlfToLocalizedResx"
+          AfterTargets="UpdateXlf"
+          BeforeTargets="CoreResGen">
+
+    <SaveXlfToResx
+      Condition="%(Identity) != ''"
+      ResxPath="%(ChildResx)"
+      XlfPath="@(XlfFiles)"
+      ReplaceWithTestString="$(LocalizedTestBuild)"
+      TestString="$(LocalizedTestString)"
+    />
+
+  </Target>
+
+    <!-- Copy paste and modify OpenSourceSign from sign.targets because it only signes the main assembly 
+  and provides no extension mechanism to add more assemblies to it
+  
+  If satellite assemblies do not get their delay sign bit switched off, the main assemblies won't load them
```

Comment:
Shamefully, no. I'll add it to my todo list, along with moving the tasks to xliff parser.